apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsConfiguration.java

```diff
@@ -0,0 +1,32 @@
+package co.elastic.apm.agent.jaxrs;
+
+import org.stagemonitor.configuration.ConfigurationOption;
+import org.stagemonitor.configuration.ConfigurationOptionProvider;
+
+/**
+ * Configuration provider for the apm jax-rs plugin
+ */
+public class JaxRsConfiguration extends ConfigurationOptionProvider {
+    private static final String JAXRS_CATEGORY = "jaxrs";
+
+    private final ConfigurationOption<Boolean> allowPathOnHierarchy = ConfigurationOption.booleanOption()
+        .key("allow_path_on_hierarchy")
+        .configurationCategory(JAXRS_CATEGORY)
+        .tags("performance")
+        .description("If set to `true`,\n" +
```

Comment:
Seems like the description is out-of sync with `configuration.asciidoc`. You can execute the test `ConfigurationExporterTest`, or just `mvn clean test` to re-generate `configuration.asciidoc`.

Reply:
Oh, I wasn't aware that the asciidoc is auto generated ^^ Will fix